src/plugins/intel_cpu/docs/fusing_disabling.md

```diff
@@ -0,0 +1,53 @@
+# Fusing disabling
+Fusing disabling is controlled by environment variables.
+
+The variables define conditions of the node which fusing should be disabled for.
+
+Fusing can be disabled for:
+
+- target node (node which we fuse into):
+
+  ```sh
+      OV_CPU_DISABLE_FUSING_NODE_TYPE='>Convolution Pooling'
+      OV_CPU_DISABLE_FUSING_NODE_TYPE='Convolution Pooling'
```

Comment:
1. It makes sense to provide a ling on the file with fusing optimizations. 
2. Can we add a comment that will explain the plugin behavior with this env variable? Like all internal optimizations that fuse any node into Convolution OR Pooling will be disabled. 
3. Can I combine `>` and  `<` inside one env variable?

Reply:
1. Will be added
2. Will add few words
3. No, you can't. Not sure if it is really necessary.

Reply:
Done